client/js/components/procedure/basicSettings/ExportSettings.vue

```diff
@@ -12,9 +12,14 @@
     <p class="u-mb">
       {{ Translator.trans('export.settings.hint') }}
     </p>
+    <dp-inline-notification
+      v-if="singleCheckedFieldId"
+      :message="Translator.trans('info.export.settings')"
+      type="info" />
     <dp-checkbox
       id="check_all"
       v-model="allChecked"
+      :disabled="allChecked === true"
```

Comment:
so, you mean that the user can click on the checkAll checkbox, but nothing happens when all checkboxes are checked? this could be confusing, don't you think so?

Reply:
roosa has confirmed that it is confusing, so my assumption is not correct here. sorry for bothering you!